src/app/components/start-load-test-widget/start-load-test-widget.component.ts

```diff
@@ -0,0 +1,139 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormControl } from '@angular/forms';
+import { LoadTestConfig } from 'src/app/models/loadTestConfig';
+
+// NgRX
+/* import { SwagPaths } from '../../state/swag-paths/swag-paths.model';
+import { POPULATE_PATHS } from '../../state/swag-paths/swag-paths.actions';
+import { Store } from '@ngrx/store';
+import { Router } from '@angular/router';
+import { ParseError } from '@angular/compiler';
+*/
+
+@Component({
+  selector: 'app-start-load-test-widget',
+  templateUrl: './start-load-test-widget.component.html',
+  styleUrls: ['./start-load-test-widget.component.scss'],
+})
+export default class StartLoadTestWidgetComponent implements OnInit {
+  time = 0;
+
+  interval;
+
+  running = false;
+
+  advance = false;
+
+  click = false;
+
+  indexValue = 0;
+
+  LFC;
+
+  Msg = '';
+
+  public MsgShowError = false;
+
+  public MsgShowSuccess = false;
+
+  private Base_Url = 'http://localhost:8083/Docutest';
+
+  AdvanceForm = new FormGroup({
```

Comment:
variables should start lowercased.

Reply:
This comment was marked as resolved, but the issue still persists. `AdvanceForm` is a field of the component, and as such should likely be named `advanceForm` instead. Additionally, I believe these corresponding fields should have type declarations. Potentially not needed for boolean and number, as I am not 100% familiar with every style guide rule. In particular however, does the `interval` field need to be of type `any`? We should ideally use an specific type.